utils/icebox-rr_graph-import.py

```diff
@@ -172,14 +172,24 @@ def __init__(self, *args, **kw):
 buffer_id = 0
 switch_buffer = ET.SubElement(
     switches, 'switch',
-    {'id': str(buffer_id), 'name': 'buffer', 'buffered': "1"},
+    {'id': str(buffer_id), 'name': 'buffer', 'buffered': "1", 'type': "mux"},
 )
 
-# Routing switch connects two nets together to form a longer wire.
+switch_buffer_sizing = ET.SubElement(
+    switch_buffer, 'sizing',
+    {'mux_trans_size': "2.32", 'buf_size': "23.54"},
```

Comment:
Could you add a comment were values these come from?

Reply:
Admittedly I don't know, they were taken from the example XML in the comment immediately above those lines...